src/app/comments/comment-form-container/comment-form/comment-form.component.css

```diff
@@ -0,0 +1,156 @@
+.comment-form{
+  margin: auto 0;
+  display: grid;
+  grid-template-columns: [line1] 30% [line2] 30% [line3] 40%;
+  grid-template-rows:  repeat(2, 1fr) 0.2fr repeat(2, 1fr) 0.2fr 1fr 0.2fr 2fr;
+  column-gap: 5px;
+}
+
+input, textarea {
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+#firstName_label {
+  grid-column: 1;
+  grid-row: 1 ;
+  align-self: end;
+}
+#firstName {
+  grid-column: 1;
+  grid-row: 2;
+}
+
+#lastName_label {
+  grid-column: 2;
+  grid-row: 1 ;
+  align-self: end;
+}
+#lastName {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+
+#email_label {
+  grid-column: 3;
+  grid-row: 1 ;
+  align-self: end;
+}
+#email {
+  grid-column: 3;
+  grid-row: 2;
+}
+
+#body_label {
+  grid-column: 1;
+  grid-row: 4;
+  align-self: end;
+}
+#body {
+  grid-column: 1 / span 3;
+  grid-row: 5;
+}
+
+.checkbox_container{
+  grid-column: 1 / span 2;
+  grid-row: 7;
+  align-self: end;
+}
+
+#acceptField{
+  margin: 0 5px;
+}
+
+#send_button{
+  grid-column: 1;
+  grid-row: 9;
+  width: 70%;
+  height: 40px;
+  align-self: end;
+}
+
+.success-message {
+  width: 90%;
+  height: 50px;
+  margin: 0 auto 30px;
+  padding: 0 20px;
+  background: #E8F3DC;
+  color: #90C555;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.success-message h2 {
+  font-size: 16px;
+  margin: 0;
+  padding: 10px;
+}
+
+.error-message{
+  width: 90%;
+  height: 50px;
+  margin: 0 auto 30px;
+  padding: 0 20px;
+  background: #ffd6e0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.red-icon{
+  color:#ff4977 ;
+}
+
+.error-message mat-icon{
+  display: inline-block;
+}
+
+.error-message h2 {
+  font-size: 16px;
+  padding: 10px;
+  color: #ff4977;
+  margin: 0;
+}
+
+.error-firstName{
+  font-size: 10px;
+  color: #ff4977;
+  grid-column: 1;
+  grid-row: 3;
+  margin-left: 5px;
+}
+
+
+.error-lastName{
+  font-size: 10px;
+  color: #ff4977;
+  grid-column: 2;
+  grid-row: 3;
+}
+
+.error-email{
+   font-size: 10px;
+   color: #ff4977;
+   grid-column: 3;
+   grid-row: 3;
+ }
+
+.error-comment{
+  font-size: 10px;
```

Comment:
try css inheritance and add to parent only. the property will be inherited by child elements